pkg/solidity-utils/test/foundry/utils/BaseBitsConfigTest.sol

```diff
@@ -5,16 +5,33 @@ pragma solidity ^0.8.24;
 import "forge-std/Test.sol";
 
 contract BaseBitsConfigTest is Test {
+    struct Bits {
+        uint256 startBit;
+        uint256 size;
+    }
+
     mapping(uint256 => bool) usedBits;
 
-    function _checkBitsUsedOnce(uint256 startBit, uint256 size) internal {
-        uint256 endBit = startBit + size;
-        for (uint256 i = startBit; i < endBit; i++) {
-            _checkBitsUsedOnce(i);
+    function _checkBitsUsedOnce(Bits[] memory bits) internal {
+        uint256 nextStartBit = 0;
+        for (uint256 i = 0; i < bits.length; i++) {
+            uint256 startBit = bits[i].startBit;
+            if (i == 0) {
+                assertEq(startBit, 0, "First bit should start at 0");
+            } else {
+                assertEq(startBit, nextStartBit, "Bits do not follow each other");
+            }
```

Comment:
Ok, it took me a while to understand why we had `if / else` as opposed to just the assertion, since `startBit` and `nextStartBit` are 0 anyways when `i == 0`.

Maybe I'd add a comment here and in the function definition explaining what this does.